### PR TITLE
accessibility: add title to the pages

### PR DESCRIPTION
**Pull request by xiaotiansu**
Everything else seems following the rules of accessibility

### CODE REVIEW

**Review by pyphilia — APPROVED** [review]
It's all good for me, but is changing the title the only change necessary for accessibility?

File: src/components/RecycleBinScreen.js [hunk 1]
```diff
@@ -60,6 +62,9 @@ const RecycleBinScreen = () => {
 
   return (
     <Main>
+      <Typography variant="h4" align="left" id={TITLE_TEXT_ID}>
+        {t('Build and share rich educational resources')}
```
Comment (pyphilia) [inline src/components/RecycleBinScreen.js:66]: It's the Recycle Bin screen, so perhaps we want a more adapted text?

**Review by pyphilia — CHANGES_REQUESTED** [review]
Thanks for the PR, but not sure about adding the titles, especially if they don't fit the screens. Could you provide screenshots? It's easier for me if you change the layout.

File: src/components/RecycleBinScreen.js (hunk 1 above)
Comment (xiaotiansu) [reply src/components/RecycleBinScreen.js:66]: image
image
image
image
Yeah, maybe we could have adaptive texts for each page.

**Comment by xiaotiansu** [comment]
I also resolved the conflict, but it seems missing on the github
image

File: src/components/main/FavoriteItems.js [hunk 2]
```diff
@@ -71,7 +73,10 @@ const FavoriteItems = () => {
 
   return (
     <Main>
-      <ItemHeader />
+      <Typography variant="h4" align="left" id={TITLE_TEXT_ID}>
+        {t('Favourite Items')}
```
Comment (pyphilia) [inline src/components/main/FavoriteItems.js:77]: I think we went for `Favorite`

File: src/config/selectors.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+export const TITLE_TEXT_ID = 'homeTitle';
```
Comment (pyphilia) [inline src/config/selectors.js:1]: An id shouldn't be duplicated, instead create one id different for each title, and the name of the selector should reflect the string content.
```
HOME_TITLE_ID = 'homeTitle'
```

**Review by pyphilia — CHANGES_REQUESTED** [review]
Thanks for the PR! Did you check the item page? And the settings of these items?

**Pull request merged**